examples/sketches/nearshore-test.geojson.json

```diff
@@ -66,7 +66,13 @@
                 "exportId": "designation",
                 "fieldType": "ComboBox",
                 "valueLabel": "Fully Protected",
-                "alternateLanguages": {}
+                "formElementId": 2987,
+                "alternateLanguages": {
+                    "pt": {
+                        "label": "Designação",
+                        "valueLabel": "Totalmente Protegido"
+                    }
+                }
```

Comment:
I updated the sketch to have the latest attribute metadata including translations.  This was probably an old sketch example before we setup the translations